lib/workers/repository/master-issue.spec.ts

```diff
@@ -29,7 +29,7 @@ async function dryRun(
 ) {
   jest.resetAllMocks();
   config.dryRun = true;
-  await masterIssue.ensureMasterIssue(config, branches);
+  await dependencyDashboard.ensureMasterIssue(config, branches);
```

Comment:
Renaming `master-issue.ts` and `ensureMasterIssue` in a later pr?

Reply:
I was intending to do that last, but forgot. Wanted to make sure that the diff didn't think it was unrelated deletion + addition, so might do a chore after merging.